eyewear_shop/data/ir_ui_view.xml

```diff
@@ -1,97 +1,173 @@
+    <field name="name">res.partner.form customization</field>
+    <field name="inherit_id" ref="base.view_partner_form"/>
+    <field name="mode">extension</field>
+    <field name="model">res.partner</field>
+    <field name="active" eval="True"/>
+    <field name="priority">360</field>
+    <field name="type">form</field>
+    <field name="arch" type="xml">
+      <data>
+        <xpath expr="//form[1]/sheet[1]/group[1]" position="after">
+          <group name="studio_group_654_1im2a6k14" string="Consultations">
+            <field name="x_visits_1">
+              <list delete="false">
+                <field name="x_sequence" widget="handle"/>
+                <field name="id" optional="show"/>
+                <field name="create_date" optional="show" string="Consultation date"/>
+                <field name="create_uid" optional="show"/>
+                <field name="x_remarks" optional="show"/>
+              </list>
+            </field>
+          </group>
+        </xpath>
+      </data>
+    </field>
+  </record>
+  <record id="x_res_partner_line_default_list_view" model="ir.ui.view">
+    <field name="name">Default list view for x_res_partner_line</field>
+    <field name="model">x_res_partner_line</field>
+    <field name="type">list</field>
+    <field name="active" eval="True"/>
+    <field name="arch" type="xml">
+      <list>
+        <field name="x_sequence" widget="handle"/>
+        <field name="x_name"/>
+      </list>
+    </field>
+  </record>
+  <record id="x_res_partner_line_default_form_view" model="ir.ui.view">
+    <field name="name">Default form view for x_res_partner_line</field>
+    <field name="model">x_res_partner_line</field>
+    <field name="type">form</field>
+    <field name="active" eval="True"/>
+    <field name="arch" type="xml">
+      <form>
+        <header/>
+        <sheet string="res_partner_line">
+          <div class="oe_title">
+            <h1>
+              <field name="x_name" required="1" placeholder="Name..."/>
+            </h1>
+          </div>
+          <group name="studio_group_8d2edb">
+            <group name="studio_group_8d2edb_left"/>
+            <group name="studio_group_8d2edb_right"/>
+          </group>
+        </sheet>
+      </form>
+    </field>
+  </record>
+  <record id="x_res_partner_line_default_search_view" model="ir.ui.view">
+    <field name="name">Default search view for x_res_partner_line</field>
+    <field name="model">x_res_partner_line</field>
+    <field name="type">search</field>
+    <field name="active" eval="True"/>
+    <field name="arch" type="xml">
+      <search>
+        <field name="x_name"/>
+      </search>
+    </field>
+  </record>
+  <record id="x_res_partner_line_default_form_view_extension" model="ir.ui.view">
+    <field name="name">Default form view for x_res_partner_line customization</field>
+    <field name="model">x_res_partner_line</field>
+    <field name="inherit_id" ref="x_res_partner_line_default_form_view"/>
```

Comment:
To be clear, I'd like to merge x_res_partner_line_default_form_view_extension into x_res_partner_line_default_form_view